views/eApostilles/applicationType.ejs

```diff
@@ -0,0 +1,150 @@
+
+                    <details class="govuk-details" data-module="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">
+                                How much are the courier fees or postage?
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <h2 class="heading-medium govuk-!-margin-top-0">Courier costs</h2>
+                            <table class="govuk-table">
+                                <thead class="govuk-table__head">
+                                    <tr class="govuk-table__row">
+                                        <th scope="col" class="govuk-table__header">Country</th>
+                                        <th scope="col" class="govuk-table__header">Price</th>
+                                    </tr>
+                                </thead>
+                                <tbody class="govuk-table__body">
+                                    <tr class="govuk-table__row">
+                                        <th scope="row" class="govuk-table__header">UK (including British Forces Post Office)</th>
+                                        <td class="govuk-table__cell">£5.50 per 1.5kg</td>
+                                    </tr>
+                                    <tr class="govuk-table__row">
+                                        <th scope="row" class="govuk-table__header">European countries (not including Albania, Armenia, Azerbaijan, Belarus, Bosnia and Herzegovina, Georgia, Liechtenstein,
+                                        Kazakhstan, Moldova, Montenegro, North Macedonia, Russia, Serbia, Turkey and Ukraine)</th>
+                                        <td class="govuk-table__cell">£14.50 per 1.5kg</td>
+                                    </tr>
+                                    <tr class="govuk-table__row">
+                                        <th scope="row" class="govuk-table__header">Rest of the world</th>
+                                        <td class="govuk-table__cell">£25 per 1.5kg</td>
+                                    </tr>
+                                </tbody>
+                            </table>
+                            <h2 class="heading-medium govuk-!-margin-top-0">If you do not want to pay courier fees</h2>
+                            <p>
+                                Your documents will be returned by courier but if you’re in the UK you can have them returned by post. You must include
+                                an A4-sized envelope with the return address written on it. Attach enough UK postage stamps to cover the return postage.
+                            </p>
+                        </div>
+                    </details>
+                </div>
+                <%if(user_data.user && !user_data.user.premiumEnabled){%>
```

Comment:
This logic means the premium journey is shown for people who don't have the flag enabled in the database.

Change it to:

<%if(user_data.user && user_data.user.premiumEnabled){%>